### DISCUSSION TITLE
Where to host

### DISCUSSION THREAD

**Discussion by tidus4400** [linkdotnet/Blog#128] [Q&A] [answered]
Hi, and MANY compliments on this egregious piece of work!
I was wondering where are you hosting it.
I've used Railway to host Rocket(Rust) websites but I am lost on where I can host a Blazor Project.
Do you have any advice?

**Comment by linkdotnet** [comment, marked as answer]
Hey @tidus4400. First of all, thank you very much for the kind words.

I personally host the application on Azure. Depending on the traffic the free tier could be sufficient enough.
There are other, cheap options, as well. In addition, I use services like Azure Blob Storage to host assets like pictures and whatnot. 

With the `SQLite` storage provider you don't necessarily need a database server in the first place, you can run it serverless. You could try this in combination with free hosting services like fly.io (I did not try them and just found them with a small Google search - so I don't have an educated opinion on how it works). That should give you a free-running version of it.

**Reply by tidus4400** [reply]
Thanks a lot for the advices! I'll try Azure as well then, it will be a good training as in the Company I'm working we're migrating everything there as well :)

**Reply by linkdotnet** [reply]
Nice. If you have a Visual Studio Subscription you have some "free" quota each month. Depending on your needs you can run everything under 5$ a month.
Only for things like custom domain name and stuff you might have to pay a higher tier web app, which costs roughly 30$/month.
